package.json

```diff
@@ -47,9 +48,9 @@
   "author": "Igor Mucsicska <mucsi_96@yahoo.com>",
   "dependencies": {
     "@types/debug": "^4.1.5",
-    "@types/node-fetch": "^2.5.4",
+    "@types/request": "^2.48.4",
     "debug": "^4.1.1",
-    "node-fetch": "^2.6.0"
+    "request": "^2.88.0"
```

Comment:
Request looks much more stable. Now BrowserStack is green every time.

Reply:
Sounds good.